hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/TransactionChecker.java

```diff
@@ -123,15 +125,15 @@ public class TransactionChecker {
     public TransactionChecker(
             @MaxSignedTxnSize final int maxSignedTxnSize,
             @NodeSelfId @NonNull final AccountID nodeAccount,
-            @NonNull final GlobalDynamicProperties dynamicProperties,
+            @NonNull final ConfigProvider configProvider,
```

Comment:
I think it is cleaner, if we pass the configuration as a parameter. Can be done in a separate PR.